yt_dlp/extractor/jiocinema.py

```diff
@@ -0,0 +1,324 @@
+
+        media_metadata['showName'] = media_metadata_response['showName']
+        media_metadata['title'] = media_metadata_response['fullTitle']
+        media_metadata['shortTitle'] = media_metadata_response['shortTitle']
+        media_metadata['shortSynopsis'] = media_metadata_response['shortSynopsis']
+        media_metadata['fullSynopsis'] = media_metadata_response['fullSynopsis']
+        media_metadata['showName'] = media_metadata_response['showName']
+        media_metadata['season'] = media_metadata_response['season']
+        media_metadata['episode'] = media_metadata_response['episode']
+        media_metadata['multiTrackAudioEnabled'] = media_metadata_response['multiTrackAudioEnabled']
+        media_metadata['introStart'] = media_metadata_response['introStart']
+        media_metadata['introEnd'] = media_metadata_response['introEnd']
+        media_metadata['recapStart'] = media_metadata_response['recapStart']
+        media_metadata['recapEnd'] = media_metadata_response['recapEnd']
+        media_metadata['creditStart'] = media_metadata_response['creditStart']
+        media_metadata['creditEnd'] = media_metadata_response['creditEnd']
+        media_metadata['is4KSupported'] = media_metadata_response['is4KSupported']
+        media_metadata['is1080PSupported'] = media_metadata_response['is1080PSupported']
+        media_metadata['isDolbySupported'] = media_metadata_response['isDolbySupported']
+        media_metadata['hasSubtitles'] = media_metadata_response['hasSubtitles']
+        media_metadata['subtitles'] = media_metadata_response['subtitles']
+
+        return media_metadata
+
+    def _get_stream_url(self, auth_token, media_id):
+        stream_endpoint = 'https://apis-jiovoot.voot.com/playbackjv/v4/{media_id}'
+
+        stream_endpoint = stream_endpoint.format(media_id=media_id)
+
+        headers = {
+            'authority': 'apis-jiovoot.voot.com',
+            'accept': 'application/json, text/plain, */*',
+            'accept-language': 'en-IN,en-GB;q=0.9,en-US;q=0.8,en;q=0.7,hi;q=0.6',
+            'accesstoken': auth_token,
+            'appname': 'RJIL_JioCinema',
+            'content-type': 'application/json',
+            'deviceid': self.deviceId,
+            'dnt': '1',
+            'origin': 'https://www.jiocinema.com',
+            'referer': 'https://www.jiocinema.com/',
+            'sec-ch-ua': '"Google Chrome";v="119", "Chromium";v="119", "Not?A_Brand";v="24"',
+            'sec-ch-ua-mobile': '?0',
+            'sec-ch-ua-platform': '"macOS"',
+            'sec-fetch-dest': 'empty',
+            'sec-fetch-mode': 'cors',
+            'sec-fetch-site': 'cross-site',
+            'uniqueid': self.deviceId,
+            'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36',
+            'versioncode': '2311030',
+            'x-platform': 'androidweb',
+            'x-platform-token': 'web'
+        }
```

Comment:
common headers between here and `_get_auth_token` can be deduped

Reply:
Fixed. Headers are now used only for 1 API.